### PR TITLE
Introduce ScalarValue.Null

### PR DESCRIPTION
**Pull request by sungam3r**
Saves some allocations.

### CODE REVIEW

File: src/Serilog/Events/ScalarValue.cs [hunk 1]
```diff
@@ -19,6 +19,11 @@ namespace Serilog.Events;
 /// </summary>
 public class ScalarValue : LogEventPropertyValue
 {
+    /// <summary>
+    /// Scalar value representing <see langword="null"/>.
+    /// </summary>
+    public static readonly ScalarValue Null = new(null);
```
Comment (sungam3r) [inline src/Serilog/Events/ScalarValue.cs:25]: or 
```suggestion
    public static ScalarValue Null { get; }= new(null);
```

**Review by nblumhardt — APPROVED** [review]
Nice! Seems like a low-risk win 👍 

My only thought was whether there'd be any advantage to keeping the static property entirely internal, but seems reasonable to expose it and encourage similar usage in other places.

`true` and `false` might also be good candidates for this treatment.

**Comment by sungam3r** [comment]
I can add cached common values in another pr.

File: src/Serilog/Events/ScalarValue.cs [hunk 2]
```diff
@@ -19,6 +19,11 @@ namespace Serilog.Events;
 /// </summary>
 public class ScalarValue : LogEventPropertyValue
 {
+    /// <summary>
+    /// Scalar value representing <see langword="null"/>.
+    /// </summary>
+    public static ScalarValue Null { get; }= new(null);
```
Comment (augustoproiete) [inline src/Serilog/Events/ScalarValue.cs:25]:
```suggestion
    public static ScalarValue Null { get; } = new(null);
```

🛃 nitpick - missing space

**Review by augustoproiete — APPROVED** [review]
LGTM

File: test/Serilog.ApprovalTests/Serilog.approved.txt [hunk 3]
```diff
@@ -409,6 +410,7 @@ namespace Serilog.Events
     {
         public SequenceValue(System.Collections.Generic.IEnumerable<Serilog.Events.LogEventPropertyValue> elements) { }
         public System.Collections.Generic.IReadOnlyList<Serilog.Events.LogEventPropertyValue> Elements { get; }
+        public static Serilog.Events.SequenceValue Empty { get; }
```
Comment (sungam3r) [inline test/Serilog.ApprovalTests/Serilog.approved.txt:413]: @nblumhardt I pulled changes from `dev` to fix approval file after merging #1785.

**Review by SimonCropp — APPROVED** [review]

**Pull request merged**